app/assets/javascripts/components/markdown-editor.js

```diff
@@ -122,6 +126,16 @@ MarkdownEditor.prototype.setTargetBlank = function (container) {
   }
 }
 
+// Bubble focus and blur events from specified element to container
+MarkdownEditor.prototype.BubbleFocus = function (element, container) {
```

Comment:
Is this method intentionally prefixed with a capital letter?

Reply:
No, it should be in camel-case.